masterlist.yaml

```diff
@@ -11836,6 +11836,39 @@ plugins:
       - crc: 0x1C586D7E
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'BardsRebornStudentofSong.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47994/' ]
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Enhanced Lights and FX' ]
+        condition: 'active("EnhancedLightsandFX.esp") and not active("BardsRebornELFXpatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Enhanced Solitude SSE' ]
+        condition: 'active("Enhanced Solitude SSE.esp") and not active("BRStudentofSong EnhancedSolitude Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Realistic Lighting Overhaul SSE' ]
+        condition: 'active("RLO - Interiors.esp") and not active("BardsRebornRLOpatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Immersive Wenches' ]
+        condition: 'active("Immersive Wenches.esp") and not active("BardsRebornStudent Wenches Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Undeath Remastered' ]
+        condition: 'active("Undeath.esp") and not active("BardsReborn Undeath Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'AI Overhaul SSE' ]
+        condition: 'active("AI Overhaul.esp") and not active("BardsRebornStudent AIOverhaul Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Unofficial Skyrim Special Edition Patch' ]
+        condition: 'active("Unofficial Skyrim Special Edition Patch.esp") and not active("BardsRebornStudent USSEP Patch.esp")'
```

Comment:
The messages should be sorted alphabetically by their `subs` key value(s) if their merge keys are the same. The `SSE`s can also be removed from mod names since they're redundant, but they're not that big of a deal.

```suggestion
      - <<: *patchProvided
        subs: [ 'AI Overhaul' ]
        condition: 'active("AI Overhaul.esp") and not active("BardsRebornStudent AIOverhaul Patch.esp")'
      - <<: *patchProvided
        subs: [ 'Enhanced Lights and FX' ]
        condition: 'active("EnhancedLightsandFX.esp") and not active("BardsRebornELFXpatch.esp")'
      - <<: *patchProvided
        subs: [ 'Enhanced Solitude' ]
        condition: 'active("Enhanced Solitude SSE.esp") and not active("BRStudentofSong EnhancedSolitude Patch.esp")'
      - <<: *patchProvided
        subs: [ 'Immersive Wenches' ]
        condition: 'active("Immersive Wenches.esp") and not active("BardsRebornStudent Wenches Patch.esp")'
      - <<: *patchProvided
        subs: [ 'Realistic Lighting Overhaul' ]
        condition: 'active("RLO - Interiors.esp") and not active("BardsRebornRLOpatch.esp")'
      - <<: *patchProvided
        subs: [ 'Undeath Remastered' ]
        condition: 'active("Undeath.esp") and not active("BardsReborn Undeath Patch.esp")'
      - <<: *patchProvided
        subs: [ 'Unofficial Skyrim Special Edition Patch' ]
        condition: 'active("Unofficial Skyrim Special Edition Patch.esp") and not active("BardsRebornStudent USSEP Patch.esp")'
```